src/libDirectoryService/DirectoryService.cpp

```diff
@@ -881,7 +882,12 @@ bool DirectoryService::UpdateDSGuardIdentity() {
     }
   }
 
-  P2PComm::GetInstance().SendMessage(peerInfo, updatedsguardidentitymessage);
+  // TODO use distributed traces from here?
+  bool inject_trace_context = false;
+
+  P2PComm::GetInstance().SendMessage(peerInfo, updatedsguardidentitymessage,
```

Comment:
can you explain what is happening here? Also `updatedsguardidentitymessage` has no camel case which seems odd

Reply:
Talk pvt.